ports/stm32/stm32_hal_msp.c

```diff
@@ -68,17 +69,25 @@ void HAL_MspInit(void) {
         ARM_MPU_ClrRegion(i);
     }
 
-    ARM_MPU_SetMemAttr(MEM_ATTR_NORMAL_NON_CACHEABLE, ARM_MPU_ATTR(
+    ARM_MPU_SetMemAttr(MEMATTR_NORMAL_NCACHE, ARM_MPU_ATTR(
                            ARM_MPU_ATTR_NON_CACHEABLE,
                            ARM_MPU_ATTR_NON_CACHEABLE));
 
+    ARM_MPU_SetMemAttr(MEMATTR_NORMAL_WB_RA_WA, ARM_MPU_ATTR(
+                           ARM_MPU_ATTR_MEMORY_(1, 1, 1, 1),
+                           ARM_MPU_ATTR_MEMORY_(1, 1, 1, 1)));
+
+    MPU->RNR = 0;
+    MPU->RBAR = ARM_MPU_RBAR(0x90000000, ARM_MPU_SH_NON, 0, 1, 0); // RO-0, NP-1, XN-0
+    MPU->RLAR = ARM_MPU_RLAR(0x9FFFFFFF, MEMATTR_NORMAL_WB_RA_WA);
```

Comment:
Adding:

```
MPU->RNR = 1;
MPU->RBAR = ARM_MPU_RBAR(0x30000000, ARM_MPU_SH_NON, 0, 1, 0); // RO-0, NP-1, XN-0
MPU->RLAR = ARM_MPU_RLAR(0x3FFFFFFF, MEMATTR_NORMAL_WB_RA_WA);
```

Provided a noticeable performance improvement for the N6 when optimizing YOLOV8 post-processing. However, this shouldn't be required per the default settings for all system memories.